SpeechResponder/eddi.json

```diff
@@ -947,20 +947,11 @@
+      "script": "{_ Jumped }\r\n{_ Triggered when you complete a jump to another system }\r\n\r\n\r\n{_ Context }\r\n{SetState('eddi_context_last_subject', 'jump')}\r\n{SetState('eddi_context_last_action', 'complete')}\r\n{SetState('eddi_context_system_name', system.name)}\r\n{SetState('eddi_context_system_system', system.name)}\r\n{SetState('eddi_context_fuel_remaining', event.fuelremaining)}\r\n{SetState('eddi_context_fuel_used', event.fuelused)}\r\n{SetState('eddi_context_wanted_list', \"\")}\r\n\r\n{set jump_count to state.eddi_context_remaining_jumps - 1}\r\n{if event.destination && event.destination != \"\":\r\n    {if event.destdistance = 0:\r\n        {ShipName()} has arrived at the {P(event.destination)} system.\r\n        {Pause(1000)}\r\n        {set next_system to RouteDetails(\"update\")}\r\n    |else:\r\n        Distance to {P(event.destination)} is {round(event.destdistance, 1)} light years\r\n        {if jump_count > 0:\r\n            , with {jump_count} {if jump_count = 1: jump |else: jumps} remaining\r\n        }.\r\n    }\r\n|else:\r\n    {if jump_count > 0:\r\n        {ShipName()} has {jump_count} {if jump_count = 1: jump |else: jumps} remaining.\r\n    }\r\n}\r\n\r\n{F('Fuel check')}\r\n\r\n{set state_report to F(\"System state report\")}\r\n{set system_missions to F(\"Mission check system\")}\r\n{set system_crimes to F(\"Crime check system\")}\r\n\r\n{if state_report || system_missions || (system_crimes && find(system_crimes, \"Warning\") < 0):\r\n    {Pause(500)}\r\n    Information:\r\n}\r\n\r\n{if state_report:\r\n    {Pause(500)}\r\n    {state_report}\r\n}\r\n{if system_missions:\r\n    {Pause(500)}\r\n    {system_missions}\r\n}\r\n{if system_crimes:\r\n    {Pause(500)}\r\n    {system_crimes}\r\n}\r\n",
```

Comment:
Not sure about subtracting 1 from the jump count. Let's check that in the live game before we release.

Reply:
Not sure about adding more to the `Jumped` event when we can just keep this within the `Next jump` script (it would trigger immediately upon plotting a course and again right after you jump into the next star system). That way, we don't need to store it with `SetState`. Because we're announcing the value as soon as we receive it we shouldn't need to subtract 1 either.

Reply:
I don't believe the speech would flow properly if the 'jumps remaining' notification was made in the `Next jump` script.

It flows much better by remaining in the `Jumped` script. Snippet of pertaining `Jumped` script...

```
{if event.destination && event.destination != "":
    {if event.destdistance = 0:
        {ShipName()} has arrived at the {P(event.destination)} system.
        {Pause(1000)}
        {set next_system to RouteDetails("update")}
    |else:
        Distance to {P(event.destination)} is {round(event.destdistance, 1)} light years
        {if jump_count > 0:
            , with {jump_count} {if jump_count = 1: jump |else: jumps} remaining
        }.
    }
|else:
    {if jump_count > 0:
        {ShipName()} has {jump_count} {if jump_count = 1: jump |else: jumps} remaining.
    }
}
```

Reply:
Without a route destination set in EDDI I currently hear nothing. With this update, I'll hear something whether or not I've explicitly set a route destination in EDDI. ;-) 

Hearing a countdown on every jump when jumping a long route may get tiresome quickly (I was plotting 200+ jumps in a route on dw2 and would have definitely disabled a script that counted down every jump). I suppose we could mitigate that by announcing the distance and jumps remaining occasionally when there are large numbers of jumps remaining and then more and more frequently as the commander approaches zero jumps remaining.

As discussed on Discord, there are alternatives to {event.destination} and {event.destdistance} that don't need to be tied to this specific event.